src/components/BurnIst.tsx

```diff
@@ -0,0 +1,127 @@
+import { AssetKind } from '@agoric/ertp';
+import { stringifyValue } from '@agoric/ui-components';
+import clsx from 'clsx';
+import { AnimatePresence, motion } from 'framer-motion';
+import { useAtomValue } from 'jotai';
+import { LoadStatus, usePublishedDatum, WalletContext } from 'lib/wallet';
+import { useContext, useMemo, useState } from 'react';
+import { displayFunctionsAtom } from 'store/app';
+import { AmountInput } from './inputs';
+
+interface Props {
+  charterOfferId: number;
+}
+
+export default function BurnIst({ charterOfferId }: Props) {
+  const walletUtils = useContext(WalletContext);
+  const { getDecimalPlaces } = useAtomValue(displayFunctionsAtom);
+
+  const { data, status } = usePublishedDatum('reserve.metrics');
+  const isLoading = status !== LoadStatus.Received;
+  const brand = data?.shortfallBalance?.brand;
+  const availableToBurn = data?.allocations?.Fee?.value ?? 0n;
+
+  const availableToBurnString = brand
+    ? 'IST to Burn (' +
+      new Intl.NumberFormat().format(
+        Number(
+          stringifyValue(
+            availableToBurn,
+            AssetKind.NAT,
+            getDecimalPlaces(brand),
+            0,
+          ),
+        ),
+      ) +
+      ' Available in Reserve)'
+    : 'Loading Reserve Balance...';
+
+  const [currentInput, setCurrentInput] = useState(null);
+
+  const [minutesUntilClose, setMinutesUntilClose] = useState(10);
+
+  const inputError = useMemo(() => {
+    if (currentInput > availableToBurn) {
+      return 'Not enough IST in reserve';
+    }
+    return '';
+  }, [currentInput, availableToBurn]);
+
+  const canMakeProposal = charterOfferId && currentInput && !inputError;
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    console.debug({ event, currentInput, minutesUntilClose });
+
+    const offer = walletUtils.makeBurnIstOffer(
```

Comment:
this reads as if the offer will burn. it's only posing a question at this point. e.g. `poseBurnIst`